API/Hardware_API.md

```diff
@@ -83,17 +83,51 @@ pin.read(function(error, value) {
 
 ### Analog pins
 
-An analog pin is a pin whose value can vary in the range between 0V and 3.3V. Pins 4 and 7 on port A and all pins on port B can read analog values (though pins 0 and 1 are pulled to 3.3V by default and are thus not recommended for this purpose). Pin 7 on port B can write an analog value.
+An analog pin is a pin whose value can vary in the range between 0V and 3.3V. 
```

Comment:
Can we make clearer that this is specified as a percentage 0-1 where 1 is 3.3V? I don't think it's explicitly stated anywhere